### DISCUSSION TITLE
Have a file that has all manifest file Installer Urls for squid caching

### DISCUSSION THREAD

**Discussion by ageekhere** [microsoft/winget-pkgs#93851] [Ideas]
Hi all, it would be useful to have an updated file which has all the manifest Installer Urls. This is useful if you are using squid cache and are using SSL/MITM to cache known downloads.

atm I had to manually get all the InstallerUrl addresses in the manifest files.

**Comment by denelon** [comment]
Installer URLs are in a state of nearly constant churn. We receive many PRs daily. It's not impossible, but it may not scale well.

**Reply by ageekhere** [reply]
Maybe a command could be added to print to console the current manifest InstallerUrl addresses, then all the user would need to do is just copy the list and there would be no need to maintain anything.

**Reply by denelon** [reply]
`winget show` displays the URL for the installer based on the system architecture, settings, and preferences. It doesn't show all installer URLs though.

**Reply by ageekhere** [reply]
So it would need something like 
`winget show --InstallerUrl addresses all`
